deep_speech_2/tests/test_augmentor.py

```diff
@@ -0,0 +1,65 @@
+"""Test augmentor class."""
+from __future__ import absolute_import
+from __future__ import division
+from __future__ import print_function
+
+import unittest
+from data_utils import audio
+from data_utils.augmentor.augmentation import AugmentationPipeline
+import random
+import numpy as np
+
+random_seed = 0
+#audio instance
+audio_data = [3.0517571e-05, -8.54492188e-04, -1.09863281e-03, -9.4604492e-04,\
+            -1.31225586e-03, -1.09863281e-03, -1.73950195e-03, -2.1057189e-03,\
+            -2.04467773e-03, -1.46484375e-03, -1.43432617e-03, -9.4604492e-04,\
+            -1.95312500e-03, -1.86157227e-03, -2.10571289e-03, -2.3193354e-03,\
+            -2.01416016e-03, -2.62451172e-03, -2.07519531e-03, -2.3803719e-03]
+audio_data = np.array(audio_data)
+samplerate = 10
+
+
+class TestAugmentor(unittest.TestCase):
+    def test_volume(self):
+        config_json = '[{"type": "volume","params": {"min_gain_dBFS": -15, '\
+        '"max_gain_dBFS": 15},"prob": 1.0}]'
+        aug_pipeline = AugmentationPipeline(
+            augmentation_config=config_json, random_seed=random_seed)
+        audio_seg = audio.AudioSegment(audio_data, samplerate)
+        aug_pipeline.transform_audio(audio_seg)
+        orig_audio = audio.AudioSegment(audio_data, samplerate)
+        self.assertFalse(np.any(audio_seg.samples == orig_audio.samples))
```

Comment:
It's meaningless to do such a "non-equal" test. Just please remove this Augmentor unitest.